mypy.ini

```diff
@@ -0,0 +1,4 @@
+[mypy]
+python_version = 3.7
```

Comment:
I have a slight preference for leaving this out, and letting CI catch regressions in the Python version matrix. But very slight :-)

Reply:
Yeah, you're probably right. One less thing to change when we eventually drop 3.7